examples/quickstart-monai/monaiexample/client_app.py

```diff
@@ -1,41 +1,74 @@
 """monaiexample: A Flower / MONAI app."""
 
 import torch
-from flwr.client import ClientApp, NumPyClient
-from flwr.common import Context
+from flwr.app import ArrayRecord, Context, Message, MetricRecord, RecordDict
+from flwr.clientapp import ClientApp
 
-from monaiexample.task import get_params, load_data, load_model, set_params, test, train
+from monaiexample.task import (
+    get_params,
+    load_data,
+    load_model,
+    set_params,
```

Comment:
```suggestion
    load_data,
    load_model,
```
We are now using the built-in mechanism to convert torch state_dicts to `ArrayRecord`. We can remove those utility functions in `task.py`